<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameDlg</class>
 <widget class="QDialog" name="RenameDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rename</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="m_oldLabel">
       <property name="text">
        <string>Old Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_old">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_newLabel">
       <property name="text">
        <string>New Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_new"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_btns">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_btns</sender>
   <signal>accepted()</signal>
   <receiver>RenameDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_btns</sender>
   <signal>rejected()</signal>
   <receiver>RenameDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
